linux_os/guide/system/software/sudo/sudo_remove_nopasswd/rule.yml

```diff
@@ -50,3 +50,8 @@ ocil: |-
     To determine if <tt>NOPASSWD</tt> has been configured for sudo, run the following command:
     <pre>$ sudo grep -ri nopasswd /etc/sudoers /etc/sudoers.d/</pre>
     The command should return no output.
+
+{{% if product == "rhel8" %}}
+platforms:
+    - not_ovirt-host
```

Comment:
Something along: `This rule is disabled for Red Hat Virtualization, it will report not applicable. RHV requires to perform operations as a root without being asked for password.`

Reply:
Thank you for all the suggestions!